src/handle_init_contract.c

```diff
@@ -33,7 +23,7 @@ void handle_init_contract(ethPluginInitContract_t *msg) {
     memset(context, 0, sizeof(*context));
 
     uint32_t selector = U4BE(msg->selector, 0);
-    if (find_selector(selector, ROCKETPOOL_SELECTORS, NUM_SELECTORS, &context->selectorIndex) !=
+    if (!find_selector(selector, ROCKETPOOL_SELECTORS, NUM_SELECTORS, &context->selectorIndex) !=
         0) {
```

Comment:
`!= 0` to remove

Reply:
And fix in the ledger DB to do in order to fix the CI

Reply:
Euufffff good catch

Reply:
The DB check is red because the variant changed from `rocketpool` to `rocket_pool`
I'm hardcoding the variant to restore the previous name